plugin/unimpaired.vim

```diff
@@ -36,8 +36,9 @@ function! s:MapNextFamily(map,cmd) abort
   let map = '<Plug>unimpaired'.toupper(a:map)
   let cmd = '".(v:count ? v:count : "")."'.a:cmd
   let end = '"<CR>'.(a:cmd == 'l' || a:cmd == 'c' ? 'zv' : '')
-  execute 'nnoremap <silent> '.map.'Previous :<C-U>exe "'.cmd.'previous'.end
-  execute 'nnoremap <silent> '.map.'Next     :<C-U>exe "'.cmd.'next'.end
+  let prevnext = printf('["\%sPrevious", "\%sNext"]', map, map)
```

Comment:
This does not really need to be a list, but having a list for `b:unimpaired_prevnext` makes sense.